cache/in-memory/src/permission.rs

```diff
@@ -97,6 +97,9 @@ impl Display for ChannelError {
                 "member (guild: {}; user: {}) is not present in the cache",
                 guild_id, user_id
             )),
+            ChannelErrorType::ParentChannelNotPresent { thread_id } => {
+                f.write_fmt(format_args!("thread {} has no parent", thread_id))
```

Comment:
I thought we avoided `format` macros due to performance

Reply:
i copied the previous implementation, i should make that change to `Display` but not for the other types, it's out of scope